app-service-environment-serviceplan-logicapp/azuredeploy.json

```diff
@@ -8,6 +8,12 @@
         "description": "The name to use for the new AppService Environment"
       }
     },
+    "location": {
```

Comment:
I'm not able to get a successful deployment with this template regardless... it looks like the RP does have a bug in handling locations, but even when I get around that the deployment fails on finding the vnet.  How do we fix that?